aten/src/ATen/core/op_registration/op_registration.cpp

```diff
@@ -80,14 +74,6 @@ void RegisterOperators::checkSchemaAndRegisterOp_(Options&& options) {
 
     checkNoDuplicateKernels_(options);
 
-    // This would have unexpected behavior since an inferred schema will not
-    // have aliasing annotations.
-    TORCH_CHECK(
```

Comment:
I think this one has to stay. Later on, you won't know anymore if the schema was inferred or not.